web/src/components/apps/NodeRow.test.js

```diff
@@ -0,0 +1,5 @@
+
+
+describe("NodeRow", () => {
+  it.todo("upgrade to react 18 and add unit tests");
+});
```

Comment:
in order to use screen and render methods from testing-library/react, we will have to upgrade react to 18.

Reply:
Let's push to get [this](https://app.shortcut.com/replicated/story/52399/upgrade-kots-to-react-18) going next week.